### PR TITLE
Fixing issue with hanging flight plan.

### PR DESCRIPTION
**Pull request by cheryly279**

### CODE REVIEW

File: src/npm-restore-modules.sh [hunk 1]
```diff
@@ -17,7 +17,8 @@ pathToModulesOnHost=${hostDest}$hostNodeModules
 if ! $ssh $host [ -d $pathToModulesOnHost ] &> /dev/null
 then
 	echo "cache does not exist at $host:${hostDest}$hostNodeModules"
-	exit $cacheExists
+	$ssh -q -O exit $host 2> /dev/null
```
Comment (cheryly279) [inline src/npm-restore-modules.sh]: We needed to exit ssh here, otherwise the connection was left hanging.

**Comment by antialias** [comment]
Please rebase on top of the latest master

**Comment by cheryly279** [comment]
@antialias Rebase complete.

**Comment by cheryly279** [comment]
@antialias  Oh wait I fucked up

**Comment by cheryly279** [comment]
@antialias Ok it's good now.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,5 @@
 build
 tmp
 node_modules
+.idea
+.idea/*
```
Comment (antialias) [inline .gitignore]: `idea` above makes `idea/*` redundant

Comment (cheryly279) [inline .gitignore]: updated

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,5 @@
 build
 tmp
 node_modules
+.idea
```
Comment (antialias) [inline .gitignore]: You'll want to add this to `.npmignore` as well so you don't accidentally end up publishing your `.idea` directory. Probably best to add these ignores in a separate commit though.

Comment (cheryly279) [inline .gitignore]: I took it out entirely. Will do as a separate PR. This PR is now just the line exiting ssh.
